lib/src/memory/csr/csr_top.dart

```diff
@@ -83,16 +83,23 @@ class CsrTop extends CsrContainer {
     }
   }
 
-  /// Create the [CsrBlock] from a configuration.
-  CsrTop({
-    required CsrTopConfig super.config,
-    required super.clk,
-    required super.reset,
-    required super.frontWrite,
-    required super.frontRead,
-    super.allowLargerRegisters,
-    this.logicalRegisterIncrement = 1,
-  }) {
+  /// Create the CsrBlock from a configuration
+  CsrTop(
+      {required CsrTopConfig super.config,
+      required super.clk,
+      required super.reset,
+      required super.frontWrite,
+      required super.frontRead,
+      super.allowLargerRegisters,
+      this.logicalRegisterIncrement = 1,
+      String? definitionName})
+      : super(
+            definitionName: definitionName ??
+                'CsrTop_A${config.minAddrBits()}_'
+                    'W${config.maxRegWidth()}_'
+                    'BO${config.blockOffsetWidth}_'
+                    'LR=${allowLargerRegisters}_'
```

Comment:
I think this `=` will get sanitized down to a `_`

Reply:
oops